### DISCUSSION TITLE
How to install this application??

### DISCUSSION THREAD

**Discussion by andrianfaa** [deaafrizal/cuyuniverse-forum#84] [Q&A] [answered]
Halo, aku Andrian, mau bertanya, untuk menginstall aplikasi ini seperti menginstall laravel pada umumnya kan yaa?? Atau berbeda?? Thank you

**Comment by deaafrizal** [comment, marked as answer]
seperti laravel pada umumnya, tapi jangan lupa install node js juga ya. gw pake versi 16.
setelah itu tinggal coba install laravel biasa, terus ketik
npm install
npm run dev --watch

thats it
